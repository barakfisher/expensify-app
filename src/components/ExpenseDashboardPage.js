import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import filtersReducer from '../reducers/filters';
import ExpensesSummary from './ExpensesSummary';


    const ExpenseDashboardPage = () => (
        <div>
            <ExpensesSummary />
            <ExpenseListFilters/>
            <ExpenseList/>
        </div>   
    );

export default ExpenseDashboardPage;

