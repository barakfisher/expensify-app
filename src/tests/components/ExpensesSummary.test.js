import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary'; 
// when testing there is no need for the connected component so we should import the named one or we will get the "Invariant Violation problem" 


test('should render few expenses summery currectly', () =>{
    const wrapper = shallow(<ExpensesSummary expenseCount={3} expensesTotal={29000}/>)
    expect(wrapper).toMatchSnapshot();
});

test('should render one expense summery currectly', () =>{
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={29000}/>)
    expect(wrapper).toMatchSnapshot();
});