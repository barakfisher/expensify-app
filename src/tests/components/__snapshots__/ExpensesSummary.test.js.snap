// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render few expenses summery currectly 1`] = `
<div>
  <h1>
    View 
    3
     
    expenses
     totaling 
    $290.00
     
  </h1>
</div>
`;

exports[`should render one expense summery currectly 1`] = `
<div>
  <h1>
    View 
    1
     
    expense
     totaling 
    $290.00
     
  </h1>
</div>
`;
